internal/dockerfile2dot/build_test.go

```diff
@@ -9,6 +9,7 @@ func TestBuildDotFile(t *testing.T) {
 	type args struct {
 		simplifiedDockerfile SimplifiedDockerfile
 		legend               bool
+		layers				 bool
```

Comment:
```suggestion
		layers               bool
```

Reply:
Unfortunately I haven't set up proper linting for this project yet — but it's already in the pipeline ...

In any case, please make sure to `gofmt` all your Go code, e.g. by running `gofmt -s -w .` — thx!

Reply:
I have formatted the project. I have used the super-linter docker image from Github and there are a couple of lint errors regarding "error checking return value", repetitive functions, etc.
![Screenshot from 2022-05-13 06-44-50](https://user-images.githubusercontent.com/35694200/168276997-c312395f-1f4e-40e0-b5f7-c132147af5b0.png)
![Screenshot from 2022-05-13 06-45-18](https://user-images.githubusercontent.com/35694200/168277015-24f233cc-8e25-4642-b252-58b71ac5468b.png)

Reply:
I've just merged #89. Please update this PR by merging or rebasing, whichever way you prefer. I've disabled the `errcheck` linting for now, will fix this later.